static/js/message_list_view.js

```diff
@@ -1445,4 +1445,76 @@ export class MessageListView {
             message_container.status_message = false;
         }
     }
+
+    update_sticky_recipient_headers() {
+        const rows_length = this._rows.size;
+        if (!rows_length) {
+            /* No headers are present */
+            return;
+        }
```

Comment:
Would this be more cleanly written as having `$headers = $table.find(".message_header");` happen early, doing `$headers.first()` as it'll be used, and checking if that exists or not? 

(Just thinking about not mixing different data sources in a single calculation, given this doesn't access `_rows` anywhere else)

Reply:
No longer relevant.